src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrEnvironment.cs

```diff
@@ -0,0 +1,8 @@
+namespace NzbDrone.Core.Notifications.Notifiarr
+{
+    public enum NotifiarrEnvironment
+    {
+        Live = 1,
```

Comment:
No use in being explicit with number assignments here, they will get auto numbered.